### DISCUSSION TITLE
能不能实现这个脚本的功能

### DISCUSSION THREAD

**Discussion by Kulahala** [the1812/Bilibili-Evolved#3313] [功能建议 / Ideas]
https://greasyfork.org/zh-CN/scripts/436341-b%E7%AB%99%E9%A6%96%E9%A1%B5%E6%8E%A8%E8%8D%90%E7%BA%AF%E4%BA%AB%E7%89%88
![Snipaste_2022-05-08_18-47-37](https://user-images.githubusercontent.com/54696830/167292713-cc57b5f5-e18d-4827-955a-dd80b52589a4.png)
主页比较喜欢这种，但不适配夜间模式，希望能出一个。顺便催更恢复动态的夜间模式

**Comment by the1812** [comment]
为啥不问问那个作者出个夜间模式

**Reply by Kulahala** [reply]
确实
